kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/create-domain.sh

```diff
@@ -181,34 +197,58 @@ function createDomainHome {
     domainPropertiesOutput="${domainOutputDir}/domain.properties"
     domainHome="/u01/oracle/user_projects/domains/${domainName}"
```

Comment:
It'd improve the sample if the domainHome came from an input property, instead of being hard coded throughout.  The same comment applies to the docker file. (I realize the original sample is hard coding the domainHome).